addons/im_livechat/static/src/legacy/public_livechat_chatbot.js

```diff
@@ -151,7 +151,14 @@ import LivechatButton from '@im_livechat/legacy/widgets/livechat_button';
         const selectedAnswer = $target.data('chatbotStepAnswerId');
 
         const redirectLink = $target.data('chatbotStepRedirectLink');
-        this.messaging.publicLivechatGlobal.chatbot.update({ isRedirecting: !!redirectLink });
+        let isRedirecting = false;
+        if (redirectLink) {
```

Comment:
You should test that the `redirectLink` [can be parsed](https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static) by URL before using it.